tools/istio-iptables/main.go

```diff
@@ -0,0 +1,538 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/joho/godotenv"
```

Comment:
New dependency ? I think we have few ways already.

Reply:
@costinm Could you give me a hint, where to find a function to read a `.env` file in istio? It's needed to ensure compatibility to the existing bash script.